lcdbackpack/lcdbackpack.py

```diff
@@ -234,6 +233,17 @@ def write(self, string):
 
         self._ser.write(str.encode(string))
 
+    def set_splash_screen(self, string, lcd_chars):
+        """
+        Sets the LCD splash screen. 
+        :param string: the text to be displayed at LCD start up
+        :param lcd_chars: the total characters of the LCD display
+        """
+        self._write_command([0x40])
```

Comment:
I would rather use a "constant" for this command as those defined from line 32 in this file.